### PR TITLE
review

### PR DESCRIPTION
**Pull request by Vojtch159**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "bugs": {
+    "url": "https://github.com/PWNFinance/style-config/issues"
+  },
+  "homepage": "https://github.com/PWNFinance/style-config#readme",
+  "peerDependencies": {
+    "eslint": "^8.11.0"
+  },
+  "dependencies": {
+    "@semantic-release/changelog": "^6.0.2",
+    "@semantic-release/git": "^10.0.1",
```
Comment (microHoffman) [inline package.json:20]: i think these + `semantic-release` should be `devDependencies

File: package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "@typescript-eslint/eslint-plugin": "^5.16.0",
+    "@typescript-eslint/parser": "^5.16.0",
+    "@vue/eslint-config-typescript": "^11.0.0",
+    "babel-eslint": "^10.1.0",
```
Comment (microHoffman) [inline package.json:24]: do we need this `babel-eslint`?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "eslint": "^8.11.0",
```
Comment (microHoffman) [inline package.json:25]: maybe move this to `devDependencies` + keep in `peerDependencies`? also similar with `prettier` and `stylelint`... however do not burn too much time on this if it won't work, just thoughts

File: .github/workflows/publish.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+jobs:
+  publish:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+      - name: "Install or retrieve from cache node & npm."
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```
[the remark this answers is not in the document]
Comment (microHoffman) [reply .github/workflows/publish.yml:17]: FYI: on frontend we use yarn 3 in the CI with this optimization: https://github.com/PWNFinance/pwn_frontend/blob/dev/.github/actions/yarn-install/action.yml ... however if you wanna stick to npm here, i guess just using this `cache` option in the `actions/setup-node@v3` step is okay:)

File: .github/workflows/publish.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Install
+        run: npm install --legacy-peer-deps
```
Comment (microHoffman) [inline .github/workflows/publish.yml:21]: what about using `npm ci` here? https://docs.npmjs.com/cli/v9/commands/npm-ci or also yarn alternative is `yarn install --immutable`

File: package.json (hunk 3 above)
Comment (Vojtch159) [reply package.json:25]: iirc I tried this when I was setting it up and it caused a lot of unnecessary problems so I will keep this the way it is if it's not causing any other problems I don't know about.

File: .github/workflows/publish.yml (hunk 5 above)
Comment (Vojtch159) [reply .github/workflows/publish.yml]: fixed

File: .cz.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
```
[the remark this answers is not in the document]
Comment (Vojtch159) [reply .cz.json]: Yea, you're right. Thanks!

File: README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+##### Stylelint
+
+Create a `stylelint.config.js` file in root directory of your project and copy this code snippet into it:
+
+```
+module.exports = {
+  extends: './node_modules/@pwnfinance/style-config/stylelint.config',
+}
+```
+
```
[the remark this answers is not in the document]
Comment (Vojtch159) [reply README.md:42]: added

**Comment by Vojtch159** [comment]
I've updated the dependencies and it resolved the stylelint issue on a test CSS file I took from our frontend repo. The stuff that stylelint was trying to do should be handled by prettier. This should fix #6.

**Review by microHoffman — APPROVED** [review]

**Pull request merged**